packages/client/src/getQuote.ts

```diff
@@ -78,8 +79,8 @@ export const getZRC20GasToken = async function (
   const foreignCoins = await this.getForeignCoins();
   const token = foreignCoins.find((foreignCoin) => {
     return (
-      foreignCoin.foreign_chain_id === chainID &&
-      foreignCoin.coin_type === "Gas"
+      String(foreignCoin.foreign_chain_id) === chainID &&
```

Comment:
I do understand why we need to do this (the new sdk type says `foreign_chain_id` is a `bigint` while our previous, custom type had it as a `string`), but I don't think we should be doing this, the real type is a string.

Reply:
This is a response from the Foreign Coins endpoint:

<img width="1286" height="825" alt="image" src="https://github.com/user-attachments/assets/2ce4f9b3-958c-46b4-a4e4-6519cd2827d1" />

Reply:
This clearly resolves as a string.

<img width="171" height="46" alt="image" src="https://github.com/user-attachments/assets/7a79d358-53bc-4b5f-9e54-1cbca8582560" />